### ISSUE TITLE
Infinite  bufferization in iOS.  

### ISSUE THREAD

**Issue by lisya** [Cigaras/IPTV.bundle#53]
Hi everyone!
Do anybody have same problem? During playing channels from list, the bufferization ring doesn't disappears. On Mac and ps4 plex client everything is ok.

**Comment by arag00rn** [comment]
Same issue for me

**Comment by Cigaras** [comment]
Before Plex developers fix their client You can use this [workaround](https://forums.plex.tv/discussion/220747/spinning-loading-circle-during-streaming-hls/p3) for now by adding `duration` in `__init__.py` file in between lines [248 and 249](https://github.com/Cigaras/IPTV.bundle/blob/master/Contents/Code/__init__.py#L248-L249), like this:
```
        items = [
            MediaObject(
                #container = Container.MP4,     # MP4, MKV, MOV, AVI
                #video_codec = VideoCodec.H264, # H264
                #audio_codec = AudioCodec.AAC,  # ACC, MP3
                #audio_channels = 2,            # 2, 6
                duration = 1000,
                parts = [
                    PartObject(
                        key = GetVideoURL(url = url)
                    )
                ],
                optimized_for_streaming = Prefs['optimized_for_streaming'] # ...
            )
        ]
```
However the progress bar will always read 1 second for the duration of the video witch might not be an issue but I do not like this solution enough to include it in the code, maybe I'll read the duration from m3u file, not sure jet...

**Comment by Cigaras** [comment]
After testing on a few clients I noticed that smarter ones recognize that it's the live stream and ignore provided duration value, looks like only iOS client have this issue, so this workaround is implemented in latest release [v1.2.6](../releases/tag/v1.2.6)/

**Comment by Cigaras** [comment]
Plex [fixed this in latest iOS player](https://forums.plex.tv/discussion/comment/1322256/#Comment_1322256) so plug-in no longer defines duration.
